languages/js-erb/config.toml

```
@@ -0,0 +1,7 @@
+name = "JS/ERB"
```

I guess since https://github.com/zed-extensions/ruby/pull/160 was merged this PR should follow the same pattern:
```suggestion
name = "JS+ERB"
```